From 897cb3427c0b72b0d881b2c4d7b4534732a7c248 Mon Sep 17 00:00:00 2001
From: Andreas Cord-Landwehr <cordlandwehr@kde.org>
Date: Mon, 4 Nov 2024 20:05:40 +0100
Subject: [PATCH] Avoid absolute build path in (private) header

Do not generate an absolute build path with host system paths into
internal headers, because that causes problems for debug symbol
unfolding.

Upstream-Status: Submitted [https://invent.kde.org/plasma/bluedevil/-/merge_requests/191]

SPDX-FileCopyrightText: Andreas Cord-Landwehr <cordlandwehr@kde.org>
SPDX-License-Identifier: MIT
Signed-off-by: Andreas Cord-Landwehr <cordlandwehr@kde.org>
---
 src/kio/bluetooth/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kio/bluetooth/CMakeLists.txt b/src/kio/bluetooth/CMakeLists.txt
index a5f6b735..cd091e58 100644
--- a/src/kio/bluetooth/CMakeLists.txt
+++ b/src/kio/bluetooth/CMakeLists.txt
@@ -3,7 +3,7 @@ set(kio_bluetooth_SRCS
 )
 
 set(kded_bluedevil.xml ${CMAKE_SOURCE_DIR}/src/interfaces/kded_bluedevil.xml)
-set_source_files_properties(${kded_bluedevil.xml} PROPERTIES INCLUDE ${CMAKE_SOURCE_DIR}/src/interfaces/kded_bluedevil_types.h)
+set_source_files_properties(${kded_bluedevil.xml} PROPERTIES INCLUDE ../../interfaces/kded_bluedevil_types.h)
 qt_add_dbus_interface(kio_bluetooth_SRCS ${kded_bluedevil.xml} kdedbluedevil)
 
 kconfig_add_kcfg_files(kio_bluetooth_SRCS GENERATE_MOC ../../settings/filereceiversettings.kcfgc)
-- 
2.43.0

